From caadebea9e541862c56951a37dccef4db5bd70f9 Mon Sep 17 00:00:00 2001
From: Patrick McCarty <patrick.mccarty@intel.com>
Date: Wed, 11 Jul 2018 16:06:16 -0700
Subject: [PATCH] Disable developer build

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 meson_options.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson_options.txt b/meson_options.txt
index 6b2bf0e..97a90c9 100644
--- a/meson_options.txt
+++ b/meson_options.txt
@@ -1 +1 @@
-option('developer_build', type : 'boolean', value : true)
\ No newline at end of file
+option('developer_build', type : 'boolean', value : false)
-- 
2.18.0

